### PR TITLE
WOO-77: Rename Merchant Token to Access Token

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

**Review by asbjornu — CHANGES_REQUESTED** [review]
The tests aren't running because `uses` and `run` can't be used in the same step. Please fix.

https://github.com/SwedbankPay/swedbank-pay-woocommerce-checkout/blob/457ab848429185830f4d09f7ee8c93d4f0348362/.github/workflows/integration-tests.yml#L39-L46

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -37,7 +37,6 @@ jobs:
           composer install
 
       - name: Run tests
-        uses: php-actions/phpunit@v1
         env:
           MERCHANT_TOKEN: ${{ secrets.MERCHANT_TOKEN }}
```
Comment (asbjornu) [inline .github/workflows/integration-tests.yml]:
```suggestion
          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by asbjornu** [comment]
The tests are failing

**Comment by aait** [comment]
> The tests are failing

Fixed

**Review by asbjornu — APPROVED** [review]

**Pull request merged**